kubedog.go

```diff
@@ -87,3 +88,11 @@ SetTestFilesPath sets the path for the test files. If SetTestFilesPath was not u
 func (kdt *Test) SetTestFilesPath(testFilesPath string) {
 	kdt.KubeContext.FilesPath = testFilesPath
 }
+
+func (kdt *Test) SetWaiterInterval(duration time.Duration) {
+	kdt.KubeContext.WaiterInterval = duration
+}
+
+func (kdt *Test) SetWaiterRetries(retries int) {
+	kdt.KubeContext.WaiterRetries = retries
+}
```

Comment:
Is this number of retries or total number of tries?

Reply:
Total number of tries. Do you want to name it Tries instead of Retries?